arcgis-ios-sdk-samples/Layers/Display subtype feature layer/README.metadata.json

```diff
@@ -30,5 +30,5 @@
         "DisplaySubtypeFeatureLayerViewController.swift",
         "DisplaySubtypeSettingsViewController.swift"
     ],
-    "title": "Display a subtype feature layer"
+    "title": "Display subtype feature layer"
```

Comment:
The title is also fixed to be consistent with its plist name as well as folder name.